fe/fe-core/src/main/java/org/apache/doris/common/Config.java

```diff
@@ -1629,6 +1629,11 @@ public class Config extends ConfigBase {
      */
     @ConfField(mutable = true, masterOnly = true)
     public static int cbo_max_statistics_job_num = 20;
+    /*
+     * the max unfinished statistics task number
+     */
+    @ConfField(mutable = true, masterOnly = true)
+    public static int cbo_max_statistics_task_num = 512;
```

Comment:
This may further control the number of statistics tasks, mainly for some tables with many columns, which divide too many statistics tasks, which may affect the performance.

Reply:
Here is a simple explanation.  we think it is easier for users to understand the limit of the number of jobs. In fact, the number of tasks is more reasonable, but users may not know how to divide tasks. after discussion, we choose the former.